practice-one/src/styles/layouts/header.css

```diff
@@ -0,0 +1,35 @@
+.main-header {
+  height: 174px;
+}
+
+.header-logo {
+  width: 180px;
+  height: 52px;
+
+  & a { display: block; }
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+}
+
+.navbar-list {
+  display: flex;
+  list-style-type: none;
```

Comment:
You have used `list-style: none` for ul tag in reset file. So you don't need to override it here